tests/udf_framework/test_create_event_handler_udf_mock.py

```diff
@@ -103,12 +103,10 @@ def test_event_handler_udf_with_two_iteration():
                    and set(mock_event_handlers.QUERY_LIST) == set(
                             list(map(lambda x: x[0], result_row[4+i:])))
 
-
-        # Comment out due to the ticket #66 - Correct listing files method for local operations
-        # assert _is_state_exist(
-        #     1, "MockEventHandlerWithTwoIterations", bucketfs_connection)
-        # assert not _is_state_exist(
-        #     0, "MockEventHandlerWithTwoIterations", bucketfs_connection)
+        assert _is_state_exist(1, "MockEventHandlerWithTwoIterations",
```

Comment:
I think, here, it would be best to compute first both function calls and store the result and variables and then merge the assert with the previous assert